app/assets/javascripts/components/stories/History/index.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import PropTypes from "prop-types";
+import HistoryItem from './HistoryItem';
+import { sortByActivityCreation } from '../../../models/beta/history';
+
+const History = ({
+  history
+}) =>
+  <div className="History">
+    {
+      sortByActivityCreation(history).reverse().map(item =>
```

Comment:
This shouldn't be inside the component. It's not performant to sort and reverse it in every re-render and it's not the component's responsibility to do it.

My suggestion: do it when the data is received (inside the method called by the action that fetches it) from the backend and save it already sorted and reversed in the store.